src/bridge/transactions/disprove.rs

```diff
@@ -85,13 +85,11 @@ impl DisproveTransaction {
     pub fn pre_sign(&mut self, context: &VerifierContext) { self.sign_input0(context); }
 
     pub fn add_input_output(&mut self, input_script_index: u32, output_script_pubkey: ScriptBuf) {
-        let output_index = 1;
-
         // Add output
-        self.tx.output[output_index] = TxOut {
+        self.tx.output.push(TxOut {
```

Comment:
Same logic as before In burn. I think we can safely limit this to just 1 output unless there's a specific reason we want to allow multiple.

Reply:
Same as in `burn`. 
Now we allow adding only one output. The output amount is predefined, user must only add the script.